deploy/docker/README.md

```diff
@@ -95,6 +95,32 @@ For example, set mqtt tcp port to 1883
 
 ``docker run -d --name emqx -e EMQX_LISTENER__TCP__EXTERNAL=1883 -p 18083:18083 -p 1883:1883 emqx/emqx:latest``
 
+#### EMQ Loaded Modules Configuration
+
+| Oprtions                 | Default            | Description                           |
+| ------------------------ | ------------------ | ------------------------------------- |
+| EMQX_LOADED_MODULES       | see content below  | default modules emqx loaded            |
+
+Default environment variable ``EMQX_LOADED_MODULES``, including
+
+- ``emqx_mod_acl_internal``
+- ``emqx_mod_presence``
+
+```bash
+# The default EMQX_LOADED_MODULES env
+EMQX_LOADED_MODULES="emqx_mod_acl_internal,emqx_mod_acl_internal"
+```
+For example, load ``emqx_mod_delayed`` and ``emqx_mod_rewrite`` modules, set it into ``EMQX_LOADED_MODULES`` and use any separator to separates it.
```

Comment:
```suggestion
For example, set ``EMQX_LOADED_MODULES=emqx_mod_delayed,emqx_mod_rewrite`` to load these two plugins.
```
When giving example, as a user, I'd prefer to have something which can copy-past